src/js/components/Nav/stories/typescript/CustomAnchor.tsx

```diff
@@ -0,0 +1,50 @@
+import React from 'react';
+import { storiesOf } from '@storybook/react';
+import isChromatic from 'storybook-chromatic/isChromatic';
+
+import { Anchor, Box, Grommet, Main, Nav } from 'grommet';
+import { grommet } from 'grommet/themes';
+import { deepMerge } from 'grommet/utils';
+
+const navItems = [
+  { label: 'HTML', href: '#' },
+  { label: 'JS', href: '#' },
+  { label: 'CSS', href: '#' },
+  { label: 'REACT', href: '#' },
+];
+
+const customTheme = deepMerge(grommet, {
+  anchor: {
+    textDecoration: 'none',
+    fontWeight: 500,
+    color: {
+      dark: 'white',
+      light: 'neutral-2',
+    },
+    hover: {
+      textDecoration: 'none',
+      fontWeight: 700,
+    },
+  },
+});
+
+const CustomAnchor = () => (
```

Comment:
Can we remove this story, now that we don't have `items`?

Reply:
done.